spec/services/facets/tagging_update_publisher_spec.rb

```diff
@@ -102,4 +108,16 @@
       end
     end
   end
+
+  def stub_get_links_request(content_id: "MY-CONTENT-ID", items: ["RELATED-LINK"])
```

Comment:
This is nice! Might be nice to add this to the test helpers in gds-api-adapters

Reply:
Turns out there's already a `stub_publishing_api_has_links` helper which stubs the same endpoint.  I'll update this PR to use that method.